tb/cocotb/soc/test_irq_ctrl.py

```
@@ -0,0 +1,238 @@
+# Register byte addresses
+#   0x00  IRQ_STATUS         RO  raw synced sources
+#   0x04  IRQ_MASK           RW  per-source enable
+#   0x08  IRQ_PENDING_MASKED RO  STATUS & MASK
+#
+# Timing note: irq_src_i goes through a 1-FF synchroniser, so after any
+# change to irq_src_i we must wait at least 2 rising edges before reading
+# status registers or sampling irq_o.
+
+import cocotb
+from cocotb.clock import Clock
+from cocotb.triggers import RisingEdge, ClockCycles
```

Flake8 (7.3.0)

[error] 'cocotb.triggers.ClockCycles' imported but unused

(F401)